.github/workflows/test_branches.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           skip_doctest: 1
           TARGET: linux
           PYENV: conda
-          PACKAGES: mpi4py openmpi
+          PACKAGES: mpi4py openmpi=4.1.2
```

Comment:
Would you consider `'openmpi>=4,!=4.1.3'`?  That way we don't "fly blind" with any new openmpi releases?